docs/zh/features/basic-concepts/fee.md

```diff
@@ -27,4 +27,4 @@ Gas价格 = 交易费 / Gas上限，代表用户为每个单位的资源消耗
     iriscli stake unbond  --from=test --shares-amount=10 --address-validator=faa1mahw6ymzvt2q3lu4pjj5pau2e8krntklgarrxy  --fee=0.12iris --gas=20000 --chain-id=<chain-id>
 ```
 
-在这个例子中执行的是完成解绑操作，这里设定的交易费上限(--fee)为0.12iris，Gas上限(--gas)为20000，Gas价格就是 0.12^18iris-atto/Gas。假设执行交易总共需要10000个Gas，那么会有 1200iris-atto 的交易费被支付给验证人节点；剩余的 0.11999iris 会被退还给用户。
+在这个例子中执行的是完成解绑操作，这里设定的交易费上限(--fee)为0.12iris，Gas上限(--gas)为20000，Gas价格就是 6000iris-nano/Gas。假设执行交易总共需要10000个Gas，那么会有 1200iris-atto 的交易费被支付给验证人节点；剩余的 0.11999iris 会被退还给用户。
```

Comment:
6000iris-nano/gas * 10000gas = 6\*10^7 iris-nano = 0.06iris = 6\*10^16 iris-atto; 0.06iris will be returned to user.